tox.ini

```diff
@@ -1,13 +1,15 @@
 [tox]
 envlist = py{27,35,36}, quality
 
+[pytest]
+addopts = --cov edx_rest_api_client --cov-report term-missing --cov-report xml
+norecursedirs = .* docs requirements
+
 [testenv]
 deps =
     -r{toxinidir}/requirements/test.txt
-
 commands =
-    nosetests --with-coverage --cover-inclusive --cover-branches --cover-html --cover-html-dir=build/coverage/html/ --cover-package=edx_rest_api_client --cover-xml --cover-xml-file=build/coverage/coverage.xml edx_rest_api_client/
```

Comment:
Do we need to add a `.coveragerc` to preserve the branch coverage option?

Reply:
Added `.coveragerc`.  Let me know if you see any issues with it.